vscode-dotnet-runtime-extension/package.json

```diff
@@ -13,7 +13,7 @@
 	"description": "Allows acquisition of the .NET runtime specifically for VS Code extension authors.",
 	"appInsightsKey": "02dc18e0-7494-43b2-b2a3-18ada5fcb522",
 	"icon": "images/dotnetIcon.png",
-	"version": "1.1.0",
+	"version": "1.2.0",
```

Comment:
Didn't you already bump this [before](https://github.com/dotnet/vscode-dotnet-runtime/pull/225/files#diff-56eb173bf3821e528c6cb0129059298ebb627e3a9864955269f63684011d4fc1R16)?

Reply:
That was to prep for releasing the new version with the workspace trust changes. Now that those have been released I'm bumping again for a future release.